config/betterquesting/DefaultQuests/Quests/MultiblockGoals-AAAAAAAAAAAAAAAAAAAADA==/EBFTime-AAAAAAAAAAAAAAAAAAAATQ==.json

```diff
@@ -31,7 +31,7 @@
+      "desc:8": "In order to get alumin(i)um ingots, you need an Electric Blast Furnace. It\u0027s another very important step towards victory, so let\u0027s get started.\n\nGetting the Mica for the heating coils might be a little tricky, but you can use a substitute for the first tier of coils. \nYou need 2 LV hatches with 4A of 32V total, or 3 LV hatches with MOOOORE power if the multiblock is not fully repaired. \nIn order to not scrape by for Nickel, check out the Novice Thaumaturgy tab.\n\n§rA few recipes produce CO2/SO2 as a fluid. You can put an output fluid hatch to the top layer of the EBF to collect it otherwise it gets auto voided. It depends on the muffler hatch how much fluid you can recover. Bart\u0027s Platinum Series recipes output fluids on the BOTTOM, so if you want to do Platinum Stuff later, remember that.\n\nMoron always says - Make Muffler Moint Mup. Or point up, anyways. Before the launch, don\u0027t forget to fix maintenance issues and put a programmed circuit if necessary.\n\nIf you want a really fancy EBF, you can add more input fluid hatches and store different fluids in each one.\n\nThe EBF also has a heat adjustment of 100K for each power tier above or below MV. You can only make LV energy hatches right now, but installing two LV energy hatches will overclock the EBF to MV power, eliminating the 100K penalty and setting the initial heat to 1801K.\n\nAs you upgrade your EBF over the course of the game, this heat adjustment will increase. Two MV energy hatches (overclocking to HV) will add +100K, two HV energy hatches (overclocking to EV) will give +200K, and so on. Coils only set the base value. This adjustment does not apply to the Volcanus.\n\nIf the pollution gets really bad, check in GT++ Multiblocks for a pollution scrubber, or upgrade your Muffler.\n"
```

Comment:
that sounds like you should use magic for nickel. maybe just advise to got to the TF

Reply:
Thank you! I addressed this by specifying which exact quest to check out in the thaum tab. I hope that is okay.